src/Compiler/ReleaseNotesBuilder.cs

```diff
@@ -33,10 +32,10 @@ public static string LabelPrefix
 
         public async Task<string> BuildReleaseNotes()
         {
-            LoadMilestones();
+            milestones = await gitHubClient.GetMilestones();
```

Comment:
Do we need the field assignment? Would it be better to pass in the milestones into the methods that need it?